src/internet_identity/tests/integration/upgrade.rs

```diff
@@ -152,7 +152,7 @@ fn ii_upgrade_should_allow_same_user_range() -> Result<(), CallError> {
     let env = env();
     let canister_id = install_ii_canister(&env, II_WASM_PREVIOUS.clone());
 
-    let stats = api::compat::stats(&env, canister_id)?;
+    let stats = api::stats(&env, canister_id)?;
```

Comment:
Does this test still test anything or do you only keep it around for the next time there is a change in the type?

Reply:
Yeah, good point. This seems like a very low signal test. Let me just delete it. 👍